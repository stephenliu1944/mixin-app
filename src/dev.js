/** 
 * 该类用于开发调试, 打包时会忽略此文件.
 */
import { module1, module2 } from './index';

var person = module1();
var age = module2.print(34);
document.querySelector('#app').innerHTML = `<h1>hello ${person.name}, ${age}!</h1><h2>DEV: ${__DEV__}</h2>`;