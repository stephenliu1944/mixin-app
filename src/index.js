export { default as module1 } from './module1';
export * as module2 from './module2';
