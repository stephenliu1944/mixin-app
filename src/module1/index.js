export default from './module1';