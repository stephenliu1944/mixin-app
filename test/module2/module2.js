import { module2 } from '../../src/index';

describe('module2', function () {
    it('print', function () {
        var info = module2.print('hello world');
        expect(info).toMatchSnapshot();
    });
});