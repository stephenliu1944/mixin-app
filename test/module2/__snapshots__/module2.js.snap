// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`module2 print 1`] = `"hello world"`;
